packages/components/pie-toggle-switch/src/index.ts

```diff
@@ -58,14 +58,13 @@ export class PieToggleSwitch extends RtlMixin(LitElement) implements ToggleSwitc
      * @private
      */
     private renderToggleSwitchLabel (): TemplateResult {
-        const { label, labelPlacement } = this;
+        const { label } = this;
 
-        if (label && labelPlacement && (labelPlacements.includes(labelPlacement))) {
```

Comment:
this check is unnecessary as the validation layer is doing that and defaulting to the default value if an invalid value is passed.